docs/1-introduction/1-3-models/1-3-1-texturing.md

```diff
@@ -0,0 +1,162 @@
+We do this by creating a **ShaderResourceView**, thankfully all the hard work is already done and we can simply fill in the few bits in a `D3D11_SHADER_RESOURCE_VIEW_DESC` by copying most of it from our `D3D11_TEXTURE2D_DESC`, the only thing new here is us telling it that the view should interpret the data as a 2D texture by specifying `D3D11_SRV_DIMENSION_TEXTURE2D`.
+
+We call `device->CreateShaderResourceView(texture.Get(), &srvDesc, &srv))` and end up with our most important piece, an `ID3D11ShaderResourceView`, at this point we no longer care about the original resource data (`WRL::ComPtr<ID3D11Texture2D> texture`) and let it go out of scope. This works because the ShaderResourceView hold an internal reference to the Texture2D so the data is not deleted until we get rid of our `ID3D11ShaderResourceView`.
+
+Before we do anything with it (other than storing it) we'll quickly go over `CreateTextureViewFromDDS()`.
+We have made DDS a special case because its not a "normal image", its a rather special and efficient image because its compressed in a way that the GPU understands allowing us to save memory and bandwidth.
+Because we have gone through the path of manually creating a texture so we know what happens under the hood a bit better, we can now allow ourselves to have all of this taken care of for us.
+
+Getting DDS in a CPU/Human readable format is a big hassle, and luckily we don't even want to, it pretty much goes to the GPU as is and it already contains the format and everything inside of it.
+All we need to do here is call `DirectX::LoadFromDDSFile()` and proceedingly `DirectX::CreateTexture()` in order to get our texture resource (`WRL::ComPtr<ID3D11Resource>`), at this point we could make our own ShaderResourceView if we want specific formats or also let DirectXTex take of it (which we will).
+We choose the latter and call `DirectX::CreateShaderResourceView` and we end up with a `ID3D11ShaderResourceView` ready to use!
+
+Now that we have our SRV ((**S**)hader(**R**)esource(**V**)iew) ready, we need to do a few more things, bind it to the shader stage, add it to the shader code and create and add a texture sampler.
+Thankfully these steps aren't overly complicated.
+
+Binding it to a shader stage (in our case the pixel shader), is as simple as getting the ID3D11DeviceContext and call `PSSetShaderResources` on it.
+(**XX**)SetShaderResources (where **XX** is "VS" (VertexShader), "PS" (PixelShader),  etc) takes a few parameters, as it can set multiple textures at once:
+
+- "StartSlot"				: The texture slot to start from (generally starting at 0).
+- "NumViews"				: The amount of textures we're setting.
+- "ppShaderResourceViews"	: The array of SRVs.
+
+If we only need to set a single texture, this is as simple as calling: `PSSetShaderResources(0, 1, srv.Get())`.
+If we call `PSSetShaderResources(2,2,srvs)` it will start at slot 2 and thus set slot 2 and 3, with the SRV's supplied.
```

Comment:
All those slotisms, I wonder if it makes sense to insert a sentence here that those can be looked at in graphics debuggers as well, like RenderDoc and that we might have to massage/backreference/ofsorts to the debugging chapter somehow... do you know what i mean?

Reply:
Inconsistent spacing in the method params

Reply:
I'll see what I can do with a renderdoc screenshot.

Fixed the spacings